Source/Core/Core/DSP/Jit/DSPEmitter.cpp

```diff
@@ -85,7 +85,7 @@ void DSPEmitter::ClearIRAMandDSPJITCodespaceReset()
   CompileDispatcher();
   m_stub_entry_point = CompileStub();
 
-  for (int i = 0x0000; i < 0x10000; i++)
+  for (size_t i = 0; i < MAX_BLOCKS; i++)
```

Comment:
I think its another one of those "no better alternative available yet" cases, but its probably a better idea to leave it until someone can clarify.
In that case, it also makes sense to leave the constant instead of using `.size()`